packages/copyable/src/Copyable.spec.tsx

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+import ClipboardJS from 'clipboard';
+import {
+  render,
+  fireEvent,
+  waitFor,
+  waitForElementToBeRemoved,
+} from '@testing-library/react';
+import { Context, jest } from '@leafygreen-ui/testing-lib';
+import Copyable from '.';
+
+describe('packages/copyable', () => {
+  test('text elements are rendered', () => {
+    const { getByText } = render(
+      <Copyable label="Label" description="Description">
+        Hello world
+      </Copyable>,
+    );
+
+    expect(getByText('Label')).toBeVisible();
+    expect(getByText('Description')).toBeVisible();
+    expect(getByText('Hello world')).toBeVisible();
+  });
+
+  describe('copy button', () => {
+    test('has tooltip when clicked', async () => {
+      await Context.within(
+        jest.spyContext(ClipboardJS, 'isSupported'),
+        async spy => {
+          spy.mockReturnValue(true);
+
+          const { getByText, queryByText } = render(
+            <Copyable label="Label" description="Description">
+              Hello world
+            </Copyable>,
+          );
+
+          expect(queryByText('Copied!')).not.toBeInTheDocument();
+
+          const copyButton = getByText('Copy');
+          expect(copyButton).toBeVisible();
+
+          await Context.within(
+            jest.spyContext(ClipboardJS.prototype, 'onClick'),
+            async spy => {
+              expect(spy).not.toHaveBeenCalled();
+              fireEvent.click(copyButton);
+              await waitFor(() => expect(spy).toHaveBeenCalled());
+            },
+          );
+
+          await waitFor(() => expect(getByText('Copied!')).toBeVisible());
+
+          // Tooltip should remain visible for a while
+          await new Promise(resolve => setTimeout(resolve, 1000));
```

Comment:
Do we have an easy way to share test utils? It'd be nice to have a sleep util.

Reply:
we have `@leafygreen-ui/testing-lib`, but theoretically I think we could just have a folder that contains unpackaged files that can be imported directly. maybe worth exploring esp. if we don't want to necessarily maintain testing-lib